Classes/Backend/ItemsProcFunc.php

```diff
@@ -0,0 +1,38 @@
+<?php
+namespace Slub\SlubEntityfacts\Backend;
+
+/***
+ * This file is part of the "slubentityfacts" Extension for TYPO3 CMS.
+ *
+ * For the full copyright and license information, please read the
+ * LICENSE.txt file that was distributed with this source code.
+ * 
+ *  (c) 2021 Tobias Kreße <Tobias.Kresse@slub-dresden.de>, SLUB Dresden
+ * 
+ ***/
+
+class  ItemsProcFunc
+{
+  /**
+  * populating the option for links
+  *
+  * @param array &$config configuration array
+  */
+  public function user_sameAsSelected(array &$config)
+  {
+    $search = $config['row']['settings.entityfacts.personality'];
+
+    //api commmunication and decoding
+    $apiAnswer = file_get_contents('http://hub.culturegraph.org/entityfacts/'.$search);
+    $apiAnswerDecode = json_decode ($apiAnswer, true);
+
+    
+    //constructing the wanted array
+    foreach ($apiAnswerDecode["sameAs"] as $item) 
+    {
+      $newItems[] = 
+      [$item['collection']['name'],$item['collection']['abbr']];
+    }
+    $config['items'] = $newItems;
+  }
+}
```

Comment:
Please configure your IDE to 

- always add a newline to the end of a file
- remove trailing spaces

You may configure your IDE to respect the .editorconfig file provided by TYPO3. Have a look at https://docs.typo3.org/m/typo3/reference-coreapi/9.5/en-us/CodingGuidelines/Introduction.html